app/src/main/res/values-xxxhdpi/booleans.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <bool name="shouldSplit">true</bool>
```

Comment:
Instead of values from xml file, decide if it is true or false from your SplitScreenManager file based on the logic provided by Chantel

Reply:
We decide based on 2 factors
1. Physical diagonal size
2. Screen density  

If I'm going to use this in `SplitScreenManager` instead, I'll have to know the density from Kotlin code and I think the current approach is better